newapi/api.conf.example

```diff
@@ -23,11 +23,11 @@ TOR_TARGETS_CONFFILE = "/etc/ooni/tor_targets.json"
 # Registration email delivery
 MAIL_SERVER = "CHANGEME"
 MAIL_PORT = 465
-MAIL_USE_SSL = True
+MAIL_USE_SSL = False
 MAIL_USERNAME = "CHANGEME"
 MAIL_PASSWORD = "CHANGEME"
 MAIL_SOURCE_ADDRESS = "contact@ooni.org"
-LOGIN_BASE_URL = "https://CHANGEME/login"
+LOGIN_BASE_URL = "https://api.ooni.io/api/v1/user_login"
```

Comment:
This is an example for 3rd parties: we should not hardcode parameters related to the ooni infra.

Reply:
It's a dependency for the tests to run properly. Should I split out the config file used for testing from the config file used as example?

Reply:
Yes: the example for 3rd party users should have "CHANGEME"s to indicate  what needs customization. The files for local integ test and github CI have specific values for those envs.